src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenThatWeWantToGenerateSupportedTargetFrameworkAlias.cs

```diff
@@ -25,24 +25,40 @@ private static List<(string targetFrameworkMoniker, string displayName)> MockSup
                 ( ".NETFramework,Version=v4.8", ".NET Framework 4.8"),
             };
 
```

Comment:
Can you add a test case where the TFM is lower case? I want to see if NuGet normalizes it, if not, you need an OrdinalIgnoreCase check as TFMs are not case sensitive.

Reply:
Good catch, thank you. I've added a test case.